### PR TITLE
feat: add welcome e-mail template for new members

### PR DESCRIPTION
**Pull request by sanjayankur31**
@OCNS/software-wg : could you please have a look at this?

### CODE REVIEW

File: extras/welcome-email-template.txt [hunk 1]
```diff
@@ -0,0 +1,33 @@
+Dear <new member>
+
+Thanks for your interest in the INCF/OCNS Software Working group.
+Welcome!
+
+The group is designed to be open to all, so there isn't any real formal
+on-boarding process. Since most of our members tend to use GitHub quite
+a bit, we thought we'd use GitHub for issues/discussion and other
```
Comment (kernfel) [inline extras/welcome-email-template.txt:8]:
```suggestion
a bit, we use GitHub for issues/discussion and other
```

File: extras/welcome-email-template.txt [hunk 2]
```diff
@@ -0,0 +1,33 @@
+Dear <new member>
+
+Thanks for your interest in the INCF/OCNS Software Working group.
```
Comment (kernfel) [inline extras/welcome-email-template.txt:3]:
```suggestion
Thank you for your interest in the INCF/OCNS Software Working group.
```

File: extras/welcome-email-template.txt [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+It's just a static GitHub pages website, so you can even open a PR yourself
+if you'd like. The source for the page is here:
+https://github.com/OCNS/SoftwareWG/blob/main/content/pages/members.rst
+
+That's pretty much it in terms of membership :)
+
+Please do let us know if there's anything else we can help with. If you
+have any ideas for tasks/projects the WG should undertake please just
+start a new discussion on our GitHub repository and folks will chime in.
```
Comment (kernfel) [inline extras/welcome-email-template.txt:33]:
```suggestion
start a new discussion on our GitHub repository and folks will chime in.

Best wishes,
<sender>
```

**Review by kernfel — APPROVED** [review]
Looks good to me apart from a few minor improvements as suggested.

**Comment by kernfel** [comment]
Hm, new to reviewing. Should I not have approved...? I only meant to approve on behalf of myself, not preclude further comments.

**Comment by sanjayankur31** [comment]
> Hm, new to reviewing. Should I not have approved...? I only meant to approve on behalf of myself, not preclude further comments.

I think when we select a team, anyone from the team can review on behalf of the team. 

I guess I could tag folks individually too :thinking:

**Comment by sanjayankur31** [comment]
we can have 15 reviewers, so I've added everyone I could there. As long as we get a few +1s, that'll be sufficient. I'll merge this at the end of the work week.

**Comment by kernfel** [comment]
In general, team is probably fine. Just need to get used to not approving right away, I guess.

**Comment by appukuttan-shailesh** [comment]
Looks good to me :+1:

**Review by brenthuisman — APPROVED** [review]
lgtm!

**Comment by jimperlewitz** [comment]
Looks good

**Comment by sanjayankur31** [comment]
We have a few approvals, so I'll go merge this now. Thanks folks :)

**Pull request merged**